### PR TITLE
Fix audit warnings

### PR DESCRIPTION
**Pull request by miguelarauj1o**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/djview4.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Djview4 < Formula
   desc "Viewer for the DjVu image format"
-  homepage "http://djvu.sourceforge.net/djview4.html"
```
Comment (dunn) [inline Formula/djview4.rb]: We still want the `/djview4.html` path.

File: Formula/ftimes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Ftimes < Formula
   desc "System baselining and evidence collection tool"
-  homepage "http://ftimes.sourceforge.net/FTimes/index.shtml"
```
Comment (dunn) [inline Formula/ftimes.rb]: Same here; keep `/FTimes/index.shtml`

File: Formula/imagesnap.rb [hunk 3]
```diff
@@ -1,12 +1,12 @@
 class Imagesnap < Formula
   desc "Tool to capture still images from an iSight or other video source"
-  homepage "http://iharder.sourceforge.net/current/macosx/imagesnap/"
+  homepage "https://iharder.sourceforge.io"
```
Comment (dunn) [inline Formula/imagesnap.rb]: Keep `/current/macosx/imagesnap/`

File: Formula/lcov.rb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 class Lcov < Formula
   desc "Graphical front-end for GCC's coverage testing tool (gcov)"
-  homepage "http://ltp.sourceforge.net/coverage/lcov.php"
+  homepage "https://ltp.sourceforge.io/"
```
Comment (dunn) [inline Formula/lcov.rb]: This just directs the user to GitHub, so let's switch to https://github.com/linux-test-project/lcov as the homepage.

File: Formula/libspectrum.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 class Libspectrum < Formula
   desc "Support library for ZX Spectrum emulator"
-  homepage "http://fuse-emulator.sourceforge.net/libspectrum.php"
+  homepage "https://fuse-emulator.sourceforge.io/"
```
Comment (dunn) [inline Formula/libspectrum.rb]: Keep `/libspectrum.php`.

File: Formula/libwmf.rb [hunk 6]
```diff
@@ -1,6 +1,6 @@
 class Libwmf < Formula
   desc "Library for converting WMF (Window Metafile Format) files"
-  homepage "http://wvware.sourceforge.net/libwmf.html"
+  homepage "https://wvware.sourceforge.io"
```
Comment (dunn) [inline Formula/libwmf.rb]: Keep `/libwmf.html`.

File: Formula/locateme.rb [hunk 7]
```diff
@@ -1,6 +1,6 @@
 class Locateme < Formula
   desc "Find your location using Apple's geolocation services"
-  homepage "http://iharder.sourceforge.net/current/macosx/locateme"
+  homepage "https://iharder.sourceforge.io"
```
Comment (dunn) [inline Formula/locateme.rb]: Keep `/current/macosx/locateme`.

File: Formula/pngcrush.rb [hunk 8]
```diff
@@ -1,6 +1,6 @@
 class Pngcrush < Formula
   desc "Optimizer for PNG files"
-  homepage "http://pmt.sourceforge.net/pngcrush/"
+  homepage "https://pmt.sourceforge.io"
```
Comment (dunn) [inline Formula/pngcrush.rb]: Keep `/pngcrush`.

File: Formula/remake.rb [hunk 9]
```diff
@@ -1,6 +1,6 @@
 class Remake < Formula
   desc "GNU Make with improved error handling, tracing, and a debugger"
-  homepage "http://bashdb.sourceforge.net/remake/"
+  homepage "https://bashdb.sourceforge.io"
```
Comment (dunn) [inline Formula/remake.rb]: Keep `/remake`.

File: Formula/ssed.rb [hunk 10]
```diff
@@ -1,6 +1,6 @@
 class Ssed < Formula
   desc "Super sed stream editor"
-  homepage "http://sed.sourceforge.net/grabbag/ssed/"
+  homepage "https://sed.sourceforge.io"
```
Comment (dunn) [inline Formula/ssed.rb]: Keep `/grabbag/ssed/`

File: Formula/imagesnap.rb (hunk 3 above)
Comment (miguelarauj1o) [inline Formula/imagesnap.rb]: @dunn, fixed. I probably forgot to check this link.
thanks.

File: Formula/lcov.rb (hunk 4 above)
Comment (miguelarauj1o) [inline Formula/lcov.rb]: @dunn, great!

File: Formula/libspectrum.rb (hunk 5 above)
Comment (miguelarauj1o) [inline Formula/libspectrum.rb]: @dunn, fixed.

File: Formula/libwmf.rb (hunk 6 above)
Comment (miguelarauj1o) [inline Formula/libwmf.rb]: @dunn, fixed.

File: Formula/locateme.rb (hunk 7 above)
Comment (miguelarauj1o) [inline Formula/locateme.rb]: @dunn, fixed.

File: Formula/pngcrush.rb (hunk 8 above)
Comment (miguelarauj1o) [inline Formula/pngcrush.rb]: @dunn, fixed.

File: Formula/remake.rb (hunk 9 above)
Comment (miguelarauj1o) [inline Formula/remake.rb]: @dunn, fixed.

File: Formula/ssed.rb (hunk 10 above)
Comment (miguelarauj1o) [inline Formula/ssed.rb]: @dunn, fixed.

File: Formula/ftimes.rb (hunk 2 above)
Comment (miguelarauj1o) [inline Formula/ftimes.rb]: @dunn, fixed.

File: Formula/djview4.rb (hunk 1 above)
Comment (miguelarauj1o) [inline Formula/djview4.rb]: @dunn, fixed.

**Comment by dunn** [comment]
Merged, thank you!

**Pull request closed without merging**